docs/developers/deploy-on-polygon.md

```diff
@@ -0,0 +1,149 @@
+
+If the event is disputed, then the following steps will be taken to bridge the disputed result back to the final arbitator, the DVM:
+1. A Polygon contract, such as a prediction market, needs a price to settle a payout. The contract expects to get this price from an optimistic oracle (“Polygon Oracle”).
+2. For some reason, a user disagrees with the price returned by the Polygon Oracle and disputes the price.
+3. The disputed price request is passed from the Polygon Oracle to a contract called the “Oracle Child Tunnel”, whose sole responsibility is to communicate with an “Oracle Root 
+4. Tunnel” on the Ethereum network. The Child Tunnel relays the dispute to Ethereum mainnet to the Root Tunnel.
+5. The Oracle Root Tunnel has special permission to request a price from the DVM, where the familiar voting and resolution process occurs amongst UMA voting token holders.
+Once the DVM has resolved a price request, the outcome of the vote is pushed to Oracle Root Tunnel. It is important to note that the DVM is not aware of which chain the request came from, nor does it need to.
+6. Like before, the Oracle Root Tunnel relays the result from the DVM to the Child Tunnel on Polygon.
+7. Finally, the Oracle Child Tunnel then sends a message back to the Polygon Oracle
+8. The outcome of the dispute is resolved.
+
+![](/img/PolygonDispute.png)
+
+## Deployment process
+
+At the end of this deployment process, you should have a deployed contract on Polygon mainnet or Mumbai testnet. The following is an overview of the deployment steps below.
+
+1. Clone the repo
+2. Install the dependencies
+3. Set your parameters
+5. Run the deployment on Mumbai testnet
+6. Run the deployment on Polygon mainnet  
+7. Connect to a financial product library  
+
+## Clone the repo
+
+1. Clone the repo:
+
+ - [launch-lsp](https://github.com/UMAprotocol/launch-lsp)
+```bash
+ git clone https://github.com/UMAprotocol/launch-lsp.git
+```
+## Install the dependencies
+
+You will need to install nodejs v14 (we recommend nvm to manage node versions) and yarn.
+
+Note: these additional dependencies are required -- you may or may not have them on your system already:
+
+- libudev
+- libusb
+
+These dependencies are installed on MacOS by installing the XCode Developer Tools. For Linux, the example ubuntu installation command for additional deps is:
+
+```bash
+sudo apt-get update && sudo apt-get install -y libudev-dev libusb-1.0-0-dev
+```
+## Setting your parameters
+
+It is important to understand that each contract requires the following parameters to be set at the point deployment. 
+
+### Long-Short-Pair parameters
+
+- `gasprice:` The gas price used for your contract deployment  
+- `url:` your node URL. In the case for Polygon deployments, you can use the Polygon Node URL.
+- `mnemonic:` Your 12 word seed phrase or mnemonic 
+- `lspCreatorAddress:` The address used in deploying your LSP. The default is set to: `0x3e665D15425fAee14eEF53B9caaa0762b243911a`
```

Comment:
I'd highly recommend removing any hardcoded contract addresses since they are changing pretty frequently. You can just point to the networks file for mumbai and polygon here. [Mumbai](https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/80001.json), [Polygon](https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/137.json)